test/types/schema.test.ts

```diff
@@ -1,10 +1,10 @@
-import { Schema, Document, SchemaDefinition, Model } from 'mongoose';
+import { Schema, Document, SchemaDefinition, Model, SchemaTypes } from 'mongoose';
 import { expectType, expectNotType, expectError } from 'tsd';
 
 enum Genre {
-  Action,
-  Adventure,
-  Comedy
+  Action = 'Action',
```

Comment:
Is this change necessary? I kinda like the default `enum` definition. Not a big deal either way since this is test code.